content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,370 @@
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_lt/data/account.account.template.csv
+    :language: csv
+    :end-at: account_account_template_1201
+
+.. example::
+  `addons/l10n_at/data/account_account_template.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_at/data/account_account_template.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_at/data/account_account_template.xml
+    :language: xml
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+
+Account groups
+--------------
+
+.. seealso::
+   :ref:`Account Group References <reference/account_group>`
+
+Account groups allows to describe structure of chart of accounts.
+
+.. example::
+  `addons/l10n_il/data/account.group.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_il/data/account.group.template.csv>`__.
+
+  .. csv-table::
+     :file: {ODOO_ABSPATH}/addons/l10n_il/data/account.group.template.csv
+     :widths: 20,20,20,20,20
+     :header-rows: 1
+
+Taxes
+-----
+
+.. seealso::
+   - :ref:`Tax References <reference/account_tax>`
+   - :doc:`/applications/finance/accounting/taxation/taxes/taxes`
+
+To add taxes you first need to specify tax groups. It has just two fields: name and country. Create file `data/account_tax_group_data.xml` and list the groups:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <data noupdate="1">
+            <record id="tax_group_tva_0" model="account.tax.group">
+                <field name="name">TVA 0%</field>
+                <field name="country_id" ref="base.ch"/>
+            </record>
+
+            ...
+        </data>
+    </odoo>
+
+.. example::
+  `addons/l10n_ch/data/account_tax_group_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/account_tax_group_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account_tax_group_data.xml
+    :language: xml
+    :start-after: <data
+    :end-before: </data>
+
+.. example::
+  `addons/l10n_uk/data/account.tax.group.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_uk/data/account.tax.group.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_uk/data/account.tax.group.csv
+    :language: csv
+
+
+Now you can add the taxes via `data/account_tax_template_data.xml` file.
+
+
+.. example::
+  `addons/l10n_ae/data/account_tax_template_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ae/data/account_tax_template_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ae/data/account_tax_template_data.xml
+    :language: xml
+    :start-at: uae_sale_tax_5_dubai
+    :end-at: </record>
+
+If some accounts should use default taxes, you can set them up in `data/account_account_template_post_data.xml`
+
+Tax Report
+----------
+
+.. seealso::
+   - :ref:`Tax Report Line References <reference/account_tax_report_line>`
+   - :doc:`/applications/finance/accounting/reporting/declarations/tax_returns`
```

Comment:
On financial reports, we explicitly state they are enterprise; we should do the same here. Only the model and the definition of its instances are community. Maybe just explaining it with 1-2 sentences here would be nice as well.